### PR TITLE
Improve on bad practices (naming, strings instead of enums, etc.)

### PR DESCRIPTION
**Pull request by Nujanauss**

### CODE REVIEW

File: mentalabMobileApi/src/main/java/com/mentalab/packets/PublishablePacket.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    protected PublishablePacket(double timeStamp) {
+        super(timeStamp);
+    }
+
+    @Override
+    public Topic getTopic() {
+        return null;
+    }
+
+    public void publish() {
```
Comment (salman2135) [inline mentalabMobileApi/src/main/java/com/mentalab/packets/PublishablePacket.java:17]: This method should be in codec. Packet classes should only have packet structure related stuff.

File: mentalabMobileApi/src/main/java/com/mentalab/ExploreDevice.java [hunk 2]
```diff
@@ -54,22 +53,22 @@ public BluetoothDevice getBluetoothDevice() {
    * @param switches List of channel switches, indicating which channels should be on and off
    * @throws InvalidCommandException
    */
-  public Future<Boolean> setActiveChannels(List<InputDataSwitch> switches)
+  public Future<Boolean> postActiveChannels(List<InputDataSwitch> switches)
```
Comment (salman2135) [inline mentalabMobileApi/src/main/java/com/mentalab/ExploreDevice.java:56]: I propose we use setActiveChannels for naming.

File: mentalabMobileApi/src/main/java/com/mentalab/ExploreDevice.java [hunk 3]
```diff
@@ -54,22 +53,22 @@ public BluetoothDevice getBluetoothDevice() {
...
       throws InvalidCommandException {
     final Command c = Command.CMD_CHANNEL_SET;
     c.setArg(generateChannelsArg(switches));
 
     return submitCommand(c);
   }
 
-  public Future<Boolean> setActiveModules(InputDataSwitch s) throws InvalidCommandException {
+  public Future<Boolean> postActiveModules(InputDataSwitch s) throws InvalidCommandException {
```
Comment (salman2135) [inline mentalabMobileApi/src/main/java/com/mentalab/ExploreDevice.java:64]: Also here- **setActiveModules**() is a better name.

File: mentalabMobileApi/src/main/java/com/mentalab/packets/PublishablePacket.java (hunk 1 above)
Comment (Nujanauss) [reply mentalabMobileApi/src/main/java/com/mentalab/packets/PublishablePacket.java:17]: Agreed - moved in the latest branch

**Pull request merged**